statsd/statsd.go

```diff
@@ -268,12 +268,27 @@ func (c *Client) Histogram(name string, value float64, tags []string, rate float
 	return c.send(name, stat, tags, rate)
 }
 
+// Decr is just Count of 1
+func (c *Client) Decr(name string, tags []string, rate float64) error {
+	return c.send(name, "-1|c", tags, rate)
+}
+
+// Incr is just Count of 1
+func (c *Client) Incr(name string, tags []string, rate float64) error {
+	return c.send(name, "1|c", tags, rate)
+}
+
 // Set counts the number of unique elements in a group.
 func (c *Client) Set(name string, value string, tags []string, rate float64) error {
 	stat := fmt.Sprintf("%s|s", value)
 	return c.send(name, stat, tags, rate)
 }
 
+// Timing sends timing information, it is an alias for TimeInMilliseconds
+func (c *Client) Timing(name string, value float64, tags []string, rate float64) error {
```

Comment:
Done!

The `Duration` is a good idea, but I still feel the usage of `TimeInMillseconds` is both not idiomatic — as evidenced by the need to convert this to millis — and confusing from an API perspective. Datadog's timers not being inherently millis but can be whatever time you like. It would save a cast and a division to make this nanos!

That being said, this does what I want now and I'm happy enough to have it merged. I of course defer to you as the maintainers!